platform/qemu.go

```diff
@@ -34,80 +45,208 @@ type Disk struct {
 	Size        string   // disk image size in bytes, optional suffixes "K", "M", "G", "T" allowed. Incompatible with BackingFile
 	BackingFile string   // raw disk image to use. Incompatible with Size.
 	DeviceOpts  []string // extra options to pass to qemu. "serial=XXXX" makes disks show up as /dev/disk/by-id/virtio-<serial>
-	ConfPath    string   // path to ignition to be able to use it with guestfs for temporary qcow2 images
 }
 
-var (
-	ErrNeedSizeOrFile  = errors.New("Disks need either Size or BackingFile specified")
-	ErrBothSizeAndFile = errors.New("Only one of Size and BackingFile can be specified")
```

Comment:
Also, there's no reason we can't support both a backing file *and* resizing (at least to a bigger size).